### PR TITLE
smbserver - add option to empty all recycle bins

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

**Comment by szaimen** [comment]
This is ready 👍

**Comment by szaimen** [comment]
I have one additional idea!

File: not-supported/smbserver.sh [hunk 1]
```diff
@@ -1289,6 +1289,102 @@ $MENU_GUIDE" "$WT_HEIGHT" "$WT_WIDTH" 4 \
...
+count=1
+while [ \$count -le $MAX_COUNT ]
+do
+    CACHE=\$(sed -n "/^#SMB\$count-start/,/^#SMB\$count-end/p" "$SMB_CONF")
+    if [ -n "\$CACHE" ]
+    then
+        SMB_PATH=\$(echo "\$CACHE" | grep "path =" | grep -oP '/.*')
+        if [ -d "\$SMB_PATH" ] && [ -d "\$SMB_PATH/.recycle/" ]
+        then
+            find "\$SMB_PATH/.recycle/" -type f -atime +2 -delete
```
Comment (enoch85) [inline not-supported/smbserver.sh:1359]: Does this delete recursively? I've tried the same thing my self, but never got it working.

Comment (szaimen) [reply not-supported/smbserver.sh]: Yes, it does. I've just tested.

**Comment by szaimen** [comment]
This is now ready 👍

**Pull request merged**